change/@ni-nimble-components-961874d3-9456-4e57-b9c6-f49f80c38843.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Note: I made this a `patch` update because there aren't any API changes, and I don't expect most clients to be impacted by this change. However, we should update the DFF boolean input when this change is picked up by SLE because they designed their boolen input to have its error icon align with the previous state of the text field.